types.ts

```diff
@@ -22,45 +22,45 @@ export interface SkillsMapping {
     emoji: string;
 }
 
-export interface Bounties {
-    id: string
-    title: string
-    slug: string
-    description: string
-    deadline: string
-    eligibility: any[]
-    status: string
-    token: string
-    rewardAmount: number
-    rewards: { first: number }
-    sponsorId: string
-    pocId: string
-    source: string
-    sourceDetails: any
-    isPublished: number
-    isFeatured: number
-    isActive: number
-    isArchived: number
-    createdAt: string
-    updatedAt: string
-    applicationLink: any
-    skills: Skills[]
-    type: string
-    requirements: any
-    totalPaymentsMade: number
-    totalWinnersSelected: number
-    isWinnersAnnounced: number
-    templateId: string
-    region: string
-    pocSocials: any
-    hackathonprize: number
-    applicationType: string
-    timeToComplete: any
-    references: any
-    winners: any
+export interface Bounty {
+    id: string;
+    title: string;
+    slug: string;
+    description: string;
+    deadline: string;
+    eligibility: any[];
+    status: string;
+    token: string;
+    rewardAmount: number;
+    rewards: { first: number };
+    sponsorId: string;
+    pocId: string;
+    source: string;
+    sourceDetails: any;
+    isPublished: number;
+    isFeatured: number;
+    isActive: number;
+    isArchived: number;
+    createdAt: string;
+    updatedAt: string;
+    applicationLink: any;
+    skills: Skill[];
+    type: string;
+    requirements: any;
+    totalPaymentsMade: number;
+    totalWinnersSelected: number;
+    isWinnersAnnounced: number;
+    templateId: string;
+    region: string;
+    pocSocials: any;
+    hackathonPrize: number;
```

Comment:
This isn't something I named, this is how it appears in the data that comes through the query

Reply:
fixed the name